wg-deployment/charter.md

```diff
@@ -0,0 +1,43 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Deployment is responsible for Kubeflow release and tooling to deploy apps from the catalog (e.g kfctl).
+
+### In scope
+
+#### Responsibilities
+
+- Provide tooling to deploy Kubeflow applications from the catalog.
+- Work with Working Group / Application Owners and Platform Owners to make the integration and release process easier.
+
+#### Code, Binaries and Services
+
+- kfctl (including operator)
+- manifests
+
+#### Cross-cutting and Externally Facing Processes
+
+##### With Application Owners
+- Co-ordinating with application OWNERs before a release to qualify application releases
+- Ensure application manifests are up to date with the latest changes.
+
+##### With Platform Owners
+- Ensure platform-specific deployment artifacts (manifests, KFDefs, kpt functions) are up-to-date.
+
+### Out of scope
+
```

Comment:
Lets add a couple things to being out of scope

* Defining or maintaining a Kubeflow distribution
* Deciding which applications or versions of applications should be included in KF distributions

Reply:
Added